.vscode/launch.json

```diff
@@ -0,0 +1,60 @@
+{
+    // Use IntelliSense to learn about possible Node.js debug attributes.
```

Comment:
AFAIK json format can't contain any comments

Reply:
@benoitf you're right. JSON specification doesn't allow comments.
Guys from the VSCode team added a separate language mode into VSCode - `jsonc` for _JSON with Comments_.
All VSCode-owned files (launch.json, tasks.json, keybindings.json etc.) are associated with that mode in VSCode.
Also, they created [jsonc-parser node module](https://www.npmjs.com/package/jsonc-parser) which takes into account the comments.